src/transformers/models/flaubert/tokenization_flaubert.py

```diff
@@ -247,7 +247,7 @@ def __init__(
         ],
         lang2id=None,
         id2lang=None,
-        do_lowercase_and_remove_accent=True,
+        do_lowercase_and_remove_accent=True,  # Keep for backward-compatibility
```

Comment:
Can we hide it in the kwargs, then in the init, pop it and send a user warning that it doesn't do anything if it's passed?